src/Kafka/KafkaInstanceDrawer/KafkaDetailsTab.tsx

```diff
@@ -63,61 +58,59 @@ export const KafkaDetailsTab: FunctionComponent<KafkaDetailsTabProps> = ({
 
   return (
     <div className="mas--details__drawer--tab-content">
-      <DetailsTabAlert expiryDate={expiryDate} />
+      {instanceType !== "standard" && expiryDate && (
+        <DetailsTabAlert expiryDate={expiryDate} />
+      )}
       <TextContent>
         <TextList component={TextListVariants.dl}>
-          {isTesting && (
-            <>
-              {instanceType === "standard" &&
-                renderTextListItem(
-                  t("common:size"),
-                  t("create-kafka-instance:streaming_size_value", {
-                    value: size,
-                  })
-                )}
-              {renderTextListItem(
-                t("ingress"),
-                t("create-kafka-instance:ingress_value", {
-                  value: ingress,
-                })
-              )}
-              {renderTextListItem(
-                t("egress"),
-                t("create-kafka-instance:egress_value", {
-                  value: egress,
-                })
-              )}
-              {renderTextListItem(
-                t("storage"),
-                t("create-kafka-instance:storage_value", {
-                  value: storage,
-                })
-              )}
-              {renderTextListItem(
-                t("partitions"),
-                t("create-kafka-instance:partitions_value", {
-                  value: maxPartitions,
-                })
-              )}
-              {renderTextListItem(
-                t("client_connections"),
-                t("create-kafka-instance:client_connections_value", {
-                  value: connections,
-                })
-              )}
-              {renderTextListItem(
-                t("connection_rate"),
-                t("create-kafka-instance:connection_rate_value", {
-                  value: connectionRate,
-                })
-              )}
-              {renderTextListItem(
-                t("message_size"),
-                t("create-kafka-instance:message_size_value", {
-                  value: messageSize,
-                })
-              )}
-            </>
+          {instanceType === "standard" &&
```

Comment:
Ideally, the namespace for this shouldn't be `create-kafka-instance` but 'kafka-details` or something like that, with its own json.

Since it's a bit of a pain, it's ok to reuse an existing namespace but please use the new `create-kafka-instance-with-size` which is the one created to support large Kafka.

Reply:
yes, will use new component `create-kafka-instance-with-size`